omod/src/main/resources/config.xml

```diff
@@ -126,34 +131,176 @@
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixEndpoint</property>
+		<defaultValue>http://localhost:5001/pix</defaultValue>
+		<description>
+			Specifies the endpoint for the Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixPort</property>
+		<defaultValue>3700</defaultValue>
+		<description>
+			Specifies port for Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.Hl7implementation</property>
+		<defaultValue>registrationcore.mpiHl7v2HttpSender</defaultValue>
+		<description>
+			Specifies which bean should be used to send the Hl7 Message. Current options are:
+			"registrationcore.mpiHl7v2HttpSender", "registrationcore.mpiHl7v2MllpSender".
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingApplication</property>
+		<defaultValue>OpenMRS</defaultValue>
+		<description>
+			Specifies the sending application in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+	
+	<globalProperty>
+		<property>${project.parent.artifactId}.birthdateEstimationStartMonth</property>
+		<defaultValue>0</defaultValue>
+		<description>
+			The month from which the birth date is estimated, with January having a value of 0
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingFacility</property>
+		<defaultValue>Demo</defaultValue>
+		<description>
+			Specifies sending Facility in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.receivingApplication</property>
+		<defaultValue>Demo MPI</defaultValue>
+		<description>
+			Specifies the receiving application in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.receivingFacility</property>
+		<defaultValue>OpenEMPI Demo</defaultValue>
+		<description>
+			Specifies sending Facility in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixIdentifierTypeUuidList</property>
+		<defaultValue>05a29f94-c0ed-11e2-94be-8c13b969e334,a5d38e09-efcb-4d91-a526-50ce1ba5011a</defaultValue>
+		<description>
+			Specifies the list of identifiers to send in a PIX query to save the patient in the MPI. Takes a list of UUID of patient identifier types - separate them with commas.
+			UUIDs can be found in the patient_identifier_type database table.
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqIdentifierTypeUuidList</property>
+		<defaultValue>05a29f94-c0ed-11e2-94be-8c13b969e334,a5d38e09-efcb-4d91-a526-50ce1ba5011a</defaultValue>
```

Comment:
Same here, you should submit all

Reply:
And this?

Reply:
as mentioned above, noted, and could the refactor be a new ticket?